cypress/e2e/basic-starter.cy.ts

```diff
@@ -198,3 +196,24 @@ describe('Node page', () => {
       });
   });
 });
+
+describe('An image using a consumer image style', () => {
+  it('has the image style source', () => {
+    const imageStyle = 'coh_medium';
+    cy.visit('/events');
+    cy.contains('Event two medium length placeholder heading.')
+      .should('contain.text', 'Event two medium length placeholder heading.')
+      .should('be.visible')
+      .click();
+    cy.get('h1')
+      .should('be.visible')
+      .should('contain.text', 'Event two medium length placeholder heading.');
+    cy.get('.media__content')
+      .find('img')
+      .should(
+        'have.attr',
+        'src',
+        `/_next/image?url=http%3A%2F%2Facms-demo.test%2Fsites%2Fdefault%2Ffiles%2Fstyles%2F${imageStyle}%2Fpublic%2F2021-02%2FImage-placeholder.png%3Fitok%3Dx6VCWB9W&w=3840&q=75`,
```

Comment:
Can we run a regex instead of having to assert the whole string?

Reply:
Changed to check if the src contains `sites%2Fdefault%2Ffiles%2Fstyles%2F${imageStyle}` instead. Would that suffice as well?

Reply:
My unsolicited $0.02: I'd probably just check that the link begins with `/_next/image?` and contains the machine name of the image style, rather than looking at the entire URL, or any part of the file path.

Reply:
I think we could safely check for "styles%2F${imageStyle}", because I believe that is _very_ unlikely to change and would confirm that the image style machine name exists in the right context.